### DISCUSSION TITLE
Adif Import wont work

### DISCUSSION THREAD

**Discussion by antonov124** [wavelog/wavelog#382] [Q&A] [answered]
I want to import the wsjtx adi      ,but  after input there is a http 500 error ;( , so at the moment in type all logs manually from wsjtx.   Anyone ? 
Wavelog 1.5.2

**Comment by phl0** [comment]
Can you send me a cioy of the adif file? Or at least a few lines?

**Reply by antonov124** [reply]
klar über welchen messenger? ;)

**Comment by phl0** [comment]
Telegram oder Signal. Oder Email :-)

**Comment by antonov124** [comment]
Email ging an deine github email

**Comment by phl0** [comment, marked as answer]
The issue here was that the php-zip package was not installed. Should have been shown in the debug information.
